worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/CylinderRegionSelector.java

```diff
@@ -244,7 +246,10 @@ public String getTypeName() {
         final List<Component> lines = new ArrayList<>();
 
         if (!region.getCenter().equals(Vector3.ZERO)) {
-            lines.add(TranslatableComponent.of("worldedit.selection.cylinder.info.center", TextComponent.of(region.getCenter().toString())));
+            Vector3 center = region.getCenter();
+            lines.add(TranslatableComponent.of("worldedit.selection.cylinder.info.center", TextComponent.of(center.toString())
+                    .clickEvent(ClickEvent.of(ClickEvent.Action.COPY_TO_CLIPBOARD, center.getX() + "," + center.getY() + "," + center.getZ()))
```

Comment:
Why add toCopyableString for BlockVector3 but not Vector?

Might we also want the 2D ones to acquire this ability? @wizjany @me4502

Reply:
Good question - no real reason, thought Vector3 might've been too generic to add something like this to?

Reply:
tbh I'm slightly confused as to why toCopyableString exists at all. Is there something undesirable about parens?

Reply:
Yeah, I feel that we should have it on all the vectors if we have it on any.

Also I'm unsure whether 'toCopyableString' best represents what it is, as it's mostly just a string representation of the vector

Reply:
My understanding is that the parens are undesirable for use in commands, such as `//pos x,y,z`, you don't have to manually strip the parens yourself. And I didn't want to change the default `toString` as having parens in other places can be useful?